### PR TITLE
Feature/854

### PR DESCRIPTION
**Pull request by edd19**
Inform the ticket you are solving in this pull request: #854

### CODE REVIEW

File: internship/models/internship_choice.py [hunk 1]
```diff
@@ -55,6 +55,10 @@ def search(student=None, internship_choice=None):
         queryset = queryset.filter(internship_choice=internship_choice)
         has_criteria = True
 
+    if speciality is not None:
```
Comment (htmfilho) [inline internship/models/internship_choice.py]: if speciality:

File: internship/models/internship_choice.py [hunk 2]
```diff
@@ -55,6 +55,10 @@ def search(student=None, internship_choice=None):
...
+        queryset = queryset.filter(speciality=speciality)
+        has_criteria = True
+
     if has_criteria:
```
Comment (htmfilho) [inline internship/models/internship_choice.py]: Merge it with the if above.

File: internship/templates/internship_home.html [hunk 3]
```diff
@@ -44,7 +44,7 @@
                 <div class="row">
                     <div class="col-md-2"><i class="fa fa-archive" style="font-size: 400%; color: #6699FF;"></i></div>
                     <div class="col-md-5">
-                        <h4 class="media-heading"><a href="{% url 'select_internship' 0 %}">{% trans 'choose_internships' %}</a></h4>
+                        <h4 class="media-heading"><a href="{% url 'select_internship' 1 %}">{% trans 'choose_internships' %}</a></h4>
```
Comment (htmfilho) [inline internship/templates/internship_home.html]: 1 ?

Comment (edd19) [inline internship/templates/internship_home.html]: Corrigé

File: internship/models/internship_choice.py (hunk 1 above)
Comment (edd19) [inline internship/models/internship_choice.py]: Corrigé

**Pull request merged**